src/main/resources/application.properties

```diff
@@ -0,0 +1,13 @@
+spring.application.name=Spring Todo Demo
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
+spring.datasource.username=postgres
+spring.datasource.password=password
```

Comment:
This should be fine, but this will likely cause some errors for people trying to run this locally when they don't have a postgres user named `postgres` with password `password`.